client/packages/executor/src/executionManager/execution.ts

```diff
@@ -166,7 +166,7 @@ export class Execution extends EventEmitter {
     getReadyToExecute(): SideEffect[] {
         let result: SideEffect[] = []
         for (let [_sfxId, sfx] of this.sideEffects) {
-            if (sfx.status === SfxStatus.ReadyToExecute && sfx.isBidder && sfx.phase === this.currentPhase) {
+            if (sfx.status === SfxStatus.Ready && sfx.isBidder && sfx.phase === this.currentPhase) {
```

Comment:
This shouldnt work tbh. The status is called `readyToExecute` (https://github.com/t3rn/t3rn/blob/development/client/packages/sdk/src/side-effects/types.ts)